python/xrprimer/visualization/keypoints/visualize_keypoints3d.py

```diff
@@ -0,0 +1,364 @@
+            Whether to plot lines according to keypoints'
+            limbs. Defaults to True.
+        background_arr (Union[np.ndarray, None], optional):
+            Background image array. Defaults to None.
+        background_dir (Union[np.ndarray, None], optional):
+            Path to the image directory for background.
+            Defaults to None.
+        background_video (Union[np.ndarray, None], optional):
+            Path to the video for background.
+            Defaults to None.
+        height (Union[int, None], optional):
+            Height of background. Defaults to None.
+        width (Union[int, None], optional):
+            Width of background. Defaults to None.
+        disable_tqdm (bool, optional):
+            Whether to disable tqdm progress bar.
+            Defaults to True.
+        logger (Union[None, str, logging.Logger], optional):
+            Logger for logging. If None, root logger will be selected.
+            Defaults to None.
+
+    Raises:
+        ValueError: Neither plot_points nor plot_lines is True.
+
+    Returns:
+        Union[np.ndarray, None]:
+            Plotted multi-frame image array or None.
+            If it's an array, its shape shall be
+            [n_frame, height, width, 3].
+    """
+    logger = get_logger(logger)
+    if not plot_points and not plot_lines:
+        logger.error('plot_points or plot_lines must be True.')
+        raise ValueError
+    # construct a projector
+    projector = OpencvProjector(camera_parameters=[camera], logger=logger)
+    n_frame = keypoints.get_frame_number()
+    n_person = keypoints.get_person_number()
+    n_kps = keypoints.get_keypoints_number()
+    # project 3d keypoints to 2d
+    projected_kps2d = projector.project(
+        points=keypoints.get_keypoints()[..., :3].reshape(
+            n_frame * n_person * n_kps, 3),
+        points_mask=keypoints.get_mask().reshape(n_frame * n_person * n_kps,
+                                                 1))
+    # concat kps2d with kps3d's confidence
+    projected_kps2d = projected_kps2d[0].reshape(n_frame, n_person, n_kps, 2)
+    projected_kps2d = np.concatenate(
+        (projected_kps2d, keypoints.get_keypoints()[..., 3:]), axis=-1)
+    keypoints2d = keypoints.clone()
+    # mask is not changed, set keypoints data is enough
+    keypoints2d.set_keypoints(
+        projected_kps2d.reshape(n_frame, n_person, n_kps, 3))
+    # call visualize_keypoints2d
+    ret_value = visualize_keypoints2d(
+        keypoints=keypoints2d,
+        output_path=output_path,
+        overwrite=overwrite,
+        return_array=return_array,
+        plot_points=plot_points,
+        plot_lines=plot_lines,
+        background_arr=background_arr,
+        background_dir=background_dir,
+        background_video=background_video,
+        height=height,
+        width=width,
+        disable_tqdm=disable_tqdm,
+        logger=logger)
+    return ret_value
+
+
+def visualize_keypoints3d_mpl(
```

Comment:
mpl stands for matplotlib? A more common abbr. seems to be `plt`, such as `import matplotlib.pyplot as plt`.

Reply:
Fixed in [f973972](https://github.com/openxrlab/xrprimer/pull/39/commits/f9739722b97083b69836a4dc395c9cbabe4e62a6).